.github/workflows/ci.yml

```diff
@@ -21,7 +21,7 @@ jobs:
 
       - uses: actions/setup-node@v4
         with:
-          node-version: '20'
+          node-version-file: 'package.json'
```

Comment:
Can't find anything about `node-version-file` in the renovate docs. But if it works :shrug:

Reply:
nothing to do with renovate, but the github action

Reply:
Ups